### ISSUE TITLE
Blue font on black background is not readable.

### ISSUE THREAD

**Issue by Xantrim** [nodeschool/discussions#858]
I'm using Windows 7 and even if I change the background color of the terminal, the tutorial program always resets the background color to black. I really have trouble reading blue text on black background. Is there any way to change the colors?

**Comment by martinheidegger** [comment]
What workshopper/adventurerer/curriculum are you having problems with?

**Comment by Xantrim** [comment]
Until now I encountered this issue with javascripting, learnyounode and bytewiser.

**Comment by martinheidegger** [comment]
@Xantrim It sounds like you have misconfigured your windows terminal (I can not reproduce your problem) Maybe using  http://bliker.github.io/cmder/ will solve your issue?

**Comment by Xantrim** [comment]
The only things I changed in my terminal are these: increased width and height, changed background color to white and text color to black. I made two screenshots to show the issue:

https://imgur.com/a/tFCfe

I don't know why this happens, but it changes all the colors to the previous settings and "console.log"  is really hard to read. 

Cmder is a really nice piece of software I didn't even know about until now! Thanks for the suggestion. I can live with that and as I seem to be the only person with this problem, I'll close this issue now.
